website/content/docs/auth/azure.mdx

```diff
@@ -234,6 +234,55 @@ tool.
        https://127.0.0.1:8200/v1/auth/azure/role/dev-role
    ```
 
+## Root credential rotation
+
+If the mount is configured with credentials directly, the credential's key may be
+rotated to a Vault-generated value that is not accessible by the operator.
+This will ensure that only Vault is able to access the "root" user that Vault uses to
+manipulate dynamic & static credentials.
+
+```shell-session
+vault write -f auth/azure/rotate-root
+```
+
+### Schedule-based root credential rotation
+
+<EnterpriseAlert product="vault" />
+
+The Azure auth engine supports configuring schedule-based automatic
+credential rotation for the root credential with the
+[rotation_schedule](/vault/api-docs/auth/azure#rotation_schedule) field.
+For example:
+
+```shell-session
+$ vault write auth/azure/config \
+    rotation_schedule="0 * * * SAT"
```

Comment:
Should we make it more obvious that this is not a complete example of configuring the mount? Something like ellipses maybe?

```suggestion
    rotation_schedule="0 * * * SAT"
    # ...
```

Reply:
We can, although this is also acceptable as a configuration to send, assuming you've already configured the backend.

Reply:
i added ellipses to the example requests to indicate that more fields can/should exist if needed/appropriate